### ISSUE TITLE
Chest glitch 

### ISSUE THREAD

**Issue by Tolo0** [iTXTech/Genisys#803]
#### Issue description
Chest going invisible 
#### Steps to reproduce the issue
I'm not sure how to reproduce this issue. iv had many players complain about Chest being invisible and when they break them. Nothing comes out.
#### OS and versions
* Genisys: I'm not sure what version. it could of happend at anytime. 
* PHP: 7
* OS:

#### Crashdump, backtrace or other files
```
Paste here
```

**Comment by PeratX** [comment]
Some plugin must have been modified chunks.

**Comment by CLiu13** [comment]
Having same issue. Very sure that this is not caused by a plugin.

Happened in old versions of MCPE, and in PM as well.

**Comment by Muqsit** [comment]
It happens if you re-generate a chunk containing tiles. It's a PocketMine glitch :/

**Comment by Tolo0** [comment]
any way to stop it from happening?

**Comment by Muqsit** [comment]
Not any as of yet :/
Contact the author of the plugin and update him/her with the issue.

**Comment by Tolo0** [comment]
I have no idea if a plugin is coursing this.

**Comment by ishitatsuyuki** [comment]
At least write your current version.

**Comment by Tolo0** [comment]
but I have no idea when this started. so it could of been any version from 2 weeks ago 😅

**Comment by CLiu13** [comment]
There was an issue concerning this previously that was closed. So, we can say that this issue was confirmed to be occurring since the post of that issue.

**Comment by PeratX** [comment]
OK. So this is an old issue of PocketMine?

**Comment by CLiu13** [comment]
Yes, it is.

**Comment by Muqsit** [comment]
Yes, @PeratX . There are many more PocketMine bugs like this one :/

**Comment by gaoyichuan** [comment]
Yes, PM does have many many bugs..

**Comment by CLiu13** [comment]
Hopefully this can be fixed!

**Comment by XxDXM-hippiexX** [comment]
I had this issue for as long as i can remember..im not sure how to reproduce it either but it doesn't seem to happen often.

**Comment by dktapps** [comment]
I get this when using copy-paste on a custom WorldEdit I wrote: the pasted chests are invisible, cannot be opened and drop nothing when pasted.

**Comment by CLiu13** [comment]
Although this is a way to reproduce the problem, the problem also occurs "naturally", without plugin interference.

**Comment by Efforto** [comment]
xD让我想起了pm大箱子问题

**Comment by Driesboy** [comment]
I dont have this bug

**Comment by PeratX** [comment]
Invisible is because of the blockentity(tile) of this chest has been removed by unnormal way. That means this is not cause by server software because Genisys does not modify chunk unnormally.

**Comment by CLiu13** [comment]
Hmm. If so, what is the correct way to remove a tile?

**Comment by Muqsit** [comment]
Removing tiles is okay. Anything that doesn't reset chunk is okay. MineReset v1.0.1 doesn't have this bug.

**Comment by CLiu13** [comment]
Alright, I guess this issue can be closed.

**Comment by Muqsit** [comment]
Yup. This is directly a PocketMine issue. We might fix this in the future once we are done with our goals :)

**Comment by isoz** [comment]
I would like some attention to this, chests going invisible are becoming a frustration among my problem.

**Comment by Muqsit** [comment]
Close this issue. This is what should be happening when you spawn a chest without the tile.

**Comment by dktapps** [comment]
@Muqsit and also when tiles get deleted. I've seen this happen myself occasionally.

**Comment by JackNoordhuis** [comment]
This issue was caused by replacing the blocks in a chunk but not retaining the entities, this caused all tiles/entities in the chunk to disappear.
